src/Tonic/Generator.h

```diff
@@ -76,6 +83,17 @@ namespace Tonic {
       obj->tick(frames, context);
     }
 
+
+  //! Shortcut for wrapping with a Panner. Currently a MonoToStereoPanner but this could/should
+  // be replaced with a better panner that can handle mono or stereo.
+  Generator pan(ControlGenerator pan);
+
+  //! Shortcut for wrapping with a Panner. Currently a MonoToStereoPanner but this could/should
+  // be replaced with a better panner that can handle mono or stereo.
+  Generator pan(float pan);
+
+  void ____recordAdditionAsParameter____(){ obj->hasBeenAdded = true; }
```

Comment:
Ah, I think I see an explanation here: http://stackoverflow.com/questions/3561648/why-does-c-not-allow-inherited-friendship/3562096#3562096